data/keys

```diff
@@ -39,20 +39,27 @@ Global {
 	KeyPress = "Ctrl Mod1 Shift Right" { Actions = "SendToWorkspace Prev; GoToWorkspace Prev" }
 	KeyPress = "Ctrl Mod1 Shift Up" { Actions = "SendToWorkspace NextV; GoToWorkspace NextV" }
 	KeyPress = "Ctrl Mod1 Shift Down" { Actions = "SendToWorkspace PrevV; GoToWorkspace PrevV" }
-	KeyPress = "Mod4 F1" { Actions = "SendToWorkspace 1" }
```

Comment:
I don't use these so I can't say, but the Ctrl-Alt-Shift-<arrow> does not behave like other WMs (mostly muscle memory, I don't know if gnome still have the same keybindings). It should be consistent with Ctrl-Alt-<arrow>

Reply:
Do you mean the `SendToWorkspace <number>` actions? I can't speak for Gnome, but for example i3, a popular wm, uses Mod4+<num< for go to workspace and Mod4+Shift+<num> for send to workspace as well.

Reply:
Arrow not number, markdown renderer considered it an html tag and of course not show it, i've fixed the original comment.